code/modules/client/preference_setup/loadout/loadout_factions.dm

```diff
@@ -90,6 +90,8 @@
 	idris_labcoats["idris letterman labcoat, alt"] = /obj/item/clothing/suit/storage/toggle/labcoat/idris/letterman/alt
 	idris_labcoats["idris labcoat, long"] = /obj/item/clothing/suit/storage/toggle/longcoat/idris
 	idris_labcoats["idris windbreaker"] = /obj/item/clothing/suit/storage/toggle/idris
+	idris_labcoats["idris officer coat"] = /obj/item/clothing/suit/storage/security/officer/idris
+	idris_labcoats["idris officer coat, alt"] = /obj/item/clothing/suit/storage/security/officer/idris/alt
```

Comment:
These new coats should be in a new subcategory, that only sec can take. The items should also be named "idris security officer coat" and not just "idris officer coat".